src/commands/community/nft/integrate.ts

```diff
@@ -35,21 +35,34 @@ async function executeNftIntegrateCommand(args: string[], msg: Message) {
   }
 
   const colDetail = await checkExistRes.json()
+  if (!colDetail.data) {
+    return {
+      messageOptions: {
+        embeds: [
+          getErrorEmbed({
+            msg,
+            title: "Invalid collection",
+            description:
+              "The collection does not exist. Please choose another one.",
+          }),
+        ],
+      },
+    }
```

Comment:
```suggestion
   throw new CommandError({ message: msg, description: "..." })
```

Quăng cái lỗi này nó sẽ tự reply user + alert luôn, hiện tại cái CommandError chưa cho phép customize title nên để a add thêm vào sau

Reply:
e mới push lên lại